dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -4446,6 +4446,15 @@ Sedp::write_subscription_data_unsecure(
                  ACE_TEXT("to ParameterList\n")));
       result = DDS::RETCODE_ERROR;
     }
+
+    DDS::TypeConsistencyEnforcementQosPolicy tceqp = TheServiceParticipant->initial_TypeConsistencyEnforcementQosPolicy();
+    tceqp.ignore_member_names = true;
```

Comment:
The behavior of type assignability will need to change if `ignore_member_names` is set to true because it currently doesn't ignore member names.

Reply:
@sonndinh @ClaytonCalabrese I understand your point. There is the option to interpret the spec with an intended meaning or we can take it literally. I don't really care as long as we're consistent but I think we need to make record of the resolution of this issue. @mitza-oci Any suggestions?